### PR TITLE
feat: Extend Disable Transformation property for Android

### PR DESCRIPTION
**Pull request by ArekChr**
## Details

This Pull Request introduces the new property disableTransformation for an android platform that disables transform in the glide framework and keeps the original size of the resource.

PROPOSAL: https://github.com/Expensify/App/issues/12893#issuecomment-1446422809

### CODE REVIEW

**Comment by ArekChr** [comment]
Hey, @cristipaval, I noticed a lot of issues while running the sample app and testing this solution. The primary reference was the Expensify project while working on the patch, as I could test it well there.

The question is, would you like me to solve this problem here?
- RNFS Example android app won't build.
- In the package.json sample app, RNFS doesn't use its library but uses npm, so I can't test local changes in the sample app.

I saw in the previous PR of this fork some errors had been fixed, but final changes were made by using patch-package

Do you think I should use any PR checklist here?

**Comment by cristipaval** [comment]
Hey @ArekChr!
I don't think Sample App is in scope for now. As far as I understand, you tested already with our App and it works as expected, right?

**Comment by ArekChr** [comment]
> Hey @ArekChr!
> I don't think Sample App is in scope for now. As far as I understand, you tested already with our App and it works as expected, right?

Yes, that's right.

**Comment by cristipaval** [comment]
Alright then!

> Do you think I should use any PR checklist here?

I don't think we have a checklist for this repo. Let's make sure that the PR follows the coding style of the upstream. We're going to open the PR to the upstream repo as well. Let's do our best to get it merged there 🤞 

cc @Beamanator since I know you went through this process before. Could you please review our conversation here? 🙏

**Comment by ArekChr** [comment]
https://github.com/DylanVann/react-native-fast-image/pull/984

**Comment by ArekChr** [comment]
@cristipaval after merging this PR, we need to do some changes in the Expensify app:
1. Set `disableTransformation` to `true` in ImageView or AttachmentView 
2. In AndroidManifest, set android:largeHeap="true"hardwareAccelerated="false"
Do you want me to create the next PR, or should we wait for the decision on whether upstream will be merged or Expensify fork will be used for RNFS

**Comment by cristipaval** [comment]
According to our plan [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1678098494778559?thread_ts=1670596654.583519&cid=C01GTK53T8Q), let's link our fork in NewDot for now, we don't know how long it takes for the PR to get merged into the upstream repo.

So just open a new PR for the NewDot and add the changes mentioned by you above, together with the new rnfs dependency (our fork).

I created [this issue](https://github.com/Expensify/App/issues/16041) to watch the PR to the upstream and if it gets merged, we'll open a new PR to the App repo to replace our fork with the upstream repo.

**Review by cristipaval — APPROVED** [review]

**Pull request merged**
